etp-backend/src/main/clj/solita/etp/service/energiatodistus_validation.clj

```diff
@@ -46,7 +46,8 @@
   (assoc v (dec (count v)) new-value))
 
 (defn u-property-condition [property]
-  (when (or (str/ends-with? property ".U") (str/ends-with? property ".g-ks"))
+  (when (or (str/ends-with? (str/lower-case property) ".u")
```

Comment:
Ilman näitä toimii, mutta parempi mielestäni näin. On robustimpi jos tätä sattumalta kutsuttaisiin ilman lower-casetettuja avaimia.